#
# Based on patch provided at http://bugs.gentoo.org/show_bug.cgi?id=265750
# Modified to not depend on "#include <string.h>" being added as our
# gcc43.patch adds "#include <cstring>" instead
#
diff -Naur libofa-0.9.3.orig/examples/example.cpp libofa-0.9.3/examples/example.cpp
--- libofa-0.9.3.orig/examples/example.cpp	2009-06-09 23:18:51.885408576 -0700
+++ libofa-0.9.3/examples/example.cpp	2009-06-09 23:19:36.701158519 -0700
@@ -9,6 +9,8 @@
 
 #include "protocol.h"
 
+#include <stdio.h>
+
 AudioData* loadWaveFile(char *file);
 AudioData* loadDataUsingLAME(char *file);
 
